### ISSUE TITLE
freqtrade dont make any order

### ISSUE THREAD

**Issue by hugohfsouza** [freqtrade/freqtrade#5990]
Hello everybody, How are you?

I did the complete installation and everything seems to work correctly. But I leave it running for a few hours and freqtrade does not do any operations. I am doing the following steps:

1. I performed the strategy analysis (hyperopt)

For that I ran the following command:
`freqtrade hyperopt --hyperopt-loss SharpeHyperOptLossDaily --spaces roi stoploss trailing --strategy HourBasedStrategy --config config.json -e 100`

The result was this:
`No epochs evaluated yet, no best result`

2. I left freqtrade running

For this I executed the following command:
`freqtrade trade --strategy HourBasedStrategy`

I left freqtrade running (online) for 4 hours but there was no buy/sell order.
What am I doing wrong?
Below are the configuration and strategy files.

**config.json**
`{
    "dry_run": true,
    "ticker_interval": "5m",
    "max_open_trades": 2,
    "tradable_balance_ratio": 1,
    "stake_currency": "USDT",
    "stake_amount": 10,
    "fiat_display_currency": "USD",
    "amend_last_stake_amount": true,
    "dry_run_wallet": 10,

    "unfilledtimeout": {
        "buy": 10,
        "sell": 30
    },
    "bid_strategy": {
        "ask_last_balance": 0.0,
        "use_order_book": false,
        "order_book_top": 1,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "ask_strategy":{
        "use_order_book": false,
        "order_book_min": 1,
        "order_book_max": 1,
        "use_sell_signal": false,
        "sell_profit_only": false,
        "ignore_roi_if_buy_signal": false
    },
    "exchange": {
        "name": "binance",
        "key": "<key_here>",
        "secret": "<secret_here>",
        "ccxt_config": {"enableRateLimit": true},
        "ccxt_async_config": {
            "enableRateLimit": true,
            "rateLimit": 200
        },
        "pair_whitelist": [
            "BNB/USDT",
            "SHIB/USDT"
        ],
        "pair_blacklist": [
        ]
    },
    "pairlists": [
        {"method": "StaticPairList"}
    ],
    "edge": {
        "enabled": false,
        "process_throttle_secs": 3600,
        "calculate_since_number_of_days": 7,
        "allowed_risk": 0.25,
        "stoploss_range_min": -0.01,
        "stoploss_range_max": -0.1,
        "stoploss_range_step": -0.01,
        "minimum_winrate": 0.60,
        "minimum_expectancy": 0.20,
        "min_trade_number": 10,
        "max_trade_duration_minute": 1440,
        "remove_pumps": false
    },
    "telegram": {
        "enabled": false,
        "token": "",
        "chat_id": ""
    },
    "api_server": {
        "enabled": true,
        "enable_openapi": true,
        "listen_ip_address": "0.0.0.0",
        "listen_port": 8080,
        "verbosity": "info",
        "jwt_secret_key": "somethingrandom",
        "CORS_origins": [],
        "username": "api",
        "password": "api"
    },
    "initial_state": "running",
    "forcebuy_enable": false,
    "internals": {
        "process_throttle_secs": 5
    }
}`

** Strategy **
the file is:

`from freqtrade.strategy import IntParameter, IStrategy
from pandas import DataFrame

class HourBasedStrategy(IStrategy):
    # Buy hyperspace params:
    buy_params = {
        "buy_hour_max": 24,
        "buy_hour_min": 4,
    }

    # Sell hyperspace params:
    sell_params = {
        "sell_hour_max": 21,
        "sell_hour_min": 22,
    }

    # ROI table:
    minimal_roi = {
        "0": 0.528,
        "169": 0.113,
        "528": 0.089,
        "1837": 0
    }

    # Stoploss:
    stoploss = -0.10

    # Optimal timeframe
    timeframe = '1h'

    buy_hour_min = IntParameter(0, 24, default=1, space='buy')
    buy_hour_max = IntParameter(0, 24, default=0, space='buy')

    sell_hour_min = IntParameter(0, 24, default=1, space='sell')
    sell_hour_max = IntParameter(0, 24, default=0, space='sell')

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe['hour'] = dataframe['date'].dt.hour
        return dataframe

    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        min, max = self.buy_hour_min.value, self.buy_hour_max.value
        min = 0
        max = 24
        dataframe.loc[
            (
                (dataframe['hour'].between(min, max))
            ),
            'buy'] = 1
        return dataframe

    def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        min, max = self.sell_hour_min.value, self.sell_hour_max.value
        dataframe.loc[
            (
                (dataframe['hour'].between(min, max))
            ),
            'buy'] = 1
        return dataframe`

** My environment **

* Operating system: MAC OS
* Python Version: Python 3.9.2
* CCXT version: ccxt==1.60.86
* Freqtrade Version: freqtrade develop-e7d1630c9

**Comment by xmatthias** [comment]
Please consult the [documentation](https://www.freqtrade.io/en/stable/faq/#i-have-waited-5-minutes-why-hasnt-the-bot-made-any-trades-yet) before asking here.

For this strategy in particular, you should give it at least 24 hours.

it also seems like you didn't run backtesting, so you will not know what to expect from this strategy (especially with the modifications you've made) - and your hyperopt kindof failed (there was no result, based on the outcome).
If these 2 commands fail, what makes you think that running in live will actually make it succeed?

Best run backtesting to verify that your strategy (with your modifications) is actually working (also in combination with your configuration - which has stake-amount set below minimum tradable limits.

**Comment by stash86** [comment]
1. Increase your dry-run wallet to 35
2. Increase your stake amount to 15

Your stake/dry wallet is too small
